email-manager/src/main.rs

```diff
@@ -87,11 +82,8 @@ fn main() -> Result<()> {
     }
 
     // Check mailbox commands.
-    match mbox_arg::matches(&m)? {
-        Some(mbox_arg::Cmd::List(max_width)) => {
-            return mbox_handler::list(max_width, &mut printer, &mut imap);
-        }
-        _ => (),
+    if let Some(mbox_arg::Cmd::List(max_width)) = mbox_arg::matches(&m)? {
```

Comment:
I would prefer to keep the original version in case we add more commands in the future (plus to match the other matches style).